packages/lib-classifier/src/helpers/subjectViewers/subjectViewers.js

```diff
@@ -10,6 +10,11 @@ Object.defineProperty(subjectViewers, 'flipbook', {
   enumerable: true
 })
 
+Object.defineProperty(subjectViewers, 'jsonData', {
+  value: 'jsonData',
```

Comment:
Do we also want to add 'jsonData' to `get viewerType()` in the WorkflowConfiguration store action too?

Reply:
Good point. I think we might.